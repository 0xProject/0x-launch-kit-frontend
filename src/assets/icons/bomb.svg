<svg xmlns='http://www.w3.org/2000/svg' xmlnsXlink='http://www.w3.org/1999/xlink'
width='26mm' height='26mm' viewBox='0 0 26 26' id='svg4557'>
    <g id='layer1' transform='translate(0 -271)'>
        <image y='270.768' x='0.038' id='image5110' xlinkHref='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAIAAAAlC+aJAAAAA3NCSVQICAjb4U/gAAANyklEQVRo ge1aeVSUV5a/973v+6oKCqhiK2RRRHEFg0vi1sYFUVziEhdyHKPRLNOdSStGbKOSVtMxicaJ6WWc 7klOpo3Rdjcm0SwuEaWICyjubApiMIAGiq2W7/ve9+aPTxHTc45VSMLMOd7zTp1D1Vvu79173/29 +0CgAvx/FtLeCjysPALQpoK+D/k/BIAgct8xtD8ABEDkBLimqoica8yn4RRI+2FABEQCHJFojA2M sK0bPKRKlssdtYRS7t0c7XGGIlJCgXOmeACASgamaeO6xr6TmOjWqCLLPu3oLw4AEQCY7AYAW0Qk pfRmxQ0AsFLpbINnxcmcioYGAARVRe+MgL9oIkMEVQGAefOff+aZtLjOXd5///3vTnw3dszY9f++ nmnahJiOnQMNAJBb7cipqkQkXmCgwi/UBBGQBlmsBw8d2X/gqzFjUg0GEwC8s/ZdznlAoAUAto4d e27qpPyZU5c+/gQAEEF84LQ/uwUooYAInFOKqqJ8/fXBkydOZGYuDwsLmz//hdjOnZcvXyZJBqGp webnd7W2LthkEJBUO111KgN8sAF+3lMIkWiqzJnKNcZUdWH6q6GhIS//5tev/Hbhzp07XC7XsePH c3Nz6+scv05KWtO3v8DV45W3bjY1eTiiF9rDzxgDiCIhiuyZMTNt8qTJJpORCEJS374DBwzoGt9t 7969o0aNuHTxIgAgFRCxk7/fwl69pnTq+I8bFcvsdipKjGneWKBNtUekSACRMRWYpqjK3LnzMjIW //Wvf6uvcwiS9B8bN1ZXV61d9y4hIIoGAJBMfrIsI0BpXX36iVOHK6urXU2AoHEv0wC0URALIhUN LXlAkMU6cdLkK4VFHaKi78MoSAGBlhdeePHatdLVq94AACKISAX9U+8CVADy4PDVWxtYgFKByW4G EGSxTpo4cdTolC5xcSaTUZSkZ+fM+aHie8nkx1QGABrnBLGh3lFRcfOTTz6ZPHVKSFjYK6+8jFTU uKaD0bt54zxwD26rBREBuSpHRsUs+d2SYb8aduN62eFvj54+fbqsrKz6VjVnKgoS51qLEYSryoUL Fy5durQwfVFBwZXY2Ng6Ry0IIvjgNi2ltW6DgqT7zNKlyy5euvzGG3+wRUTeh44KVDQgFfC+VIDm gMAmpyvtX2YPGTqssLCYUPow3tvK7aeEMsUTZLHu2LFdUdjQoUPr6hwDBw9RVKWhvh4AVI1zpjCm AiAIAtEdg3OggtPZNGr06OKioqWvLcvNy9MYk4xGWVFbp0lrkgAiYYonIqKD3W7PPm6fOGHc09Om FReVuJxNNbdvaYCKLHNV7hwY2M9mi7dYiKpoqkIJBQDgXAM8mWOvuX3L8eOPTz45rENUtOx23/nV d/HdAojAtSCL9eDBg+9v2PDhhx/s3r1n6tNTx0986nx+vigZFNk9pEPk7NFPiv5BtYoSIFLRLW89 mnWkrIwIon4+oiARxLffXmMJCcmx25OSkuobGhBJy2jxWnz0OSoaAGD/ga+WLl0GALl5Zyurqvr1 HwAAVDICwJQe8ZsXL3z8scTmFbrHxf09fcEz/foBALSgN/pUW7ZuO3YsGwBEydCaMPBNe8kIAOnp i3bv+RQA9u77bO++z4IsVgCQjEagQrzFunnxwuDQsODQsPT0RatXvbFs2Yqkvv1N5oC/py/oZ7MB 4j2KJoj68X/tWmlGxhIAoKLvGHzoKoiEUltE5LnzF4JDwwYOGjJyZLLuVVQ0oCABwNKxY6eNTgGA VxdncM4/3rz5uxMnOecJiY8NTEj8w7TpcD/H1I0wcNAQR129LSISCAEvGGjL5kMQU0I1xhYtWrTv 009rbt8qKir89tvDKEggCExjHDgBCA0NyS25CoiJiX2cLvfuXXvy8s6UXS+vddReKi31N/ubCGlJ E5jGqGQ8eSInO9v+2rLXQNN8jWavASAyVTYHBA4b9qu/ffABINbWOu4kqRYKqRxczibgvHfvXlWV lc/Nnzd+XGplZWV1VRVDZKpi+Gfyyzkgrl37zozp083+ZqZ49FtbGwOgSEDTRo9OKSu7XnGjnAoS ELwvxQJqAKpHtpr9AwItjw/of+rUqdW/f/3UqVODBj5htVgsfmaVQ72q/kQ/pjEk1G7PbmxoGDt2 HHBO0Qe/8MECADBy5IijR48AIvmnPULgAJBbWDRhwBOW8LDCwqKkpKRde/bGxcXNmjWrurpqZvKI cwUFmr4XP1GCChpj3x7Nmjjpqea1vBRv84CmaQAQ26nTpk2bgHNV4/o9614Hzqlo+Pzi+dXduw3v 3atHj+4th08bnRJN8L/y8wmlXFPvq1/dncduz1nw239rXqtNASByppjMAX7+5rLr1wGRcw2YCkiR 0mZH0jQGVHhr757FEyatnvtcTnHh7VpHqNUyuGcv6qhbs32HCwki4dodsolIOFOAc02QALG4uMhq DRYlg6LIQAUvuZ3XmZhzS1CQpqlNDY3AuSRJnTp2lT3yD1WVsizf6QKAAC4kb37+ab/omMfju/SO 69jgdh86eDC7vByQoh42CABAgBuMhuioOAC4Xl4uq3JNbY1BEgMCA2tu3/JWK5+ohCRKqqp5PK6E xMciIjtcu3r1h8qbzSQMkRBEQKQEQJLO3Kw88/2NOyORGgwmjpwxrnHOgeuMyNXkvF5e3ikmZtSo UUWFhXW1tUQQTCaT9yr5BsDtdhOEMWNS6+rrjxw6qDEGSIESioRxjauyXtT8X0qbnHk8rnt/UoFS QdM0TjTZ7SkuKiwrK+s3YMCggQNdTpfL5fKpwus1AMT6Okd0dLTL7Tp5IgcFCQRCkTBVZVwBwPju Pbp0juvSpUt0dLTFajX7+4vincmdTmetw1FRUVFRWVlcUJB/7pxemaOigQEDFFWNn8yx94jvFhBo djU1+Vih9prAzZ077+PNm+Eu66J3b+Xpixbv2/fZ6dO5/EHS2NiUm3c2/9z5NW+9Hdc1HgBAEPVG BBEAL18pmDEzDXwiRd4D2LFzV2rqeECkokH/ZtiTwwsKCnfs3NU7IWHBwgWKqrpcbo+sKKrqkZWW TZaV9eveDQ+3ZWQs0cE46uqXr8gEQKQCCKJOE//4pz9v3rwFdGLblmQOKQji/gNfRcV0hLu0eeKk yYqqfvHFF2Z/c3CQ5WhWFmOMc64yTVWZpnG9KarKOf/oo/82+5vDw22nTp2WZUVlmkdWOOfbtm3X CalkNAJgevqiL7/8yicLeJeJCQLTAECgAgAwxZPUt/+2rVsJ4l/+sjE4KMhoNNb+WEMIuVF+Y86c OTe+/x7v1nYooSpjW7ZsCQ4KEimtrq4SRaGkuHjWM7NKS8vS0ma+u24dZwyQAILNFtHQ2KiHnJcR 4BUASihw5nA4evbupSfON99809/fz+2RnQ31hFJJklavXvXiiy9NmvTUNwcOuFwuAADONc4RweP2 OBsaCaWiIGSuWDHn2TnTpk49nn3MoyiapiUnJwMCUxlwPmLkiKyjWV6q7gMAXQ4dPpQ2Y6ZOHp3O Jk3TTEZjp86dGxsbTX5+jpraA59/Xl1ZFd0xJiLCxjkAIkFkTPPz84uO7VRfX693O3L4cG1NTZfY zhE2GyK5dOmy/tgxeMiQ+Pj4f2zfBojMh4cmL12NUpM5IDfvbFLf/gAw7/kXdHfPzTvbKTYu1Boc G9MxqkNkcJBl9+49nHOXyy3LikdWXC63pvGCgsKevXoHB1k6RkZF2iJCrcHbtm3Xo/lff/MyIOpH 0OuvrwQf72XeFrb0OsqUqdMyMzMHDx5kNJquXLncIcJGCDl3/sKfNmwovnYtJCT06aen+Pv5JyQk duvWtXks53Ds+LGS4qtn8nKvlpQEh4Y8N2/emJQUlbGGhoau8d1qbt/asXNXVFTU0KFDCBU0fUzb WqD5MF3z1ttZ2XaTyW/MmFTOudsj6xvpcrk55y++9BIABFms8+Y/v3b9e2vXv/fq4gzdaGFhYbdu 327OCfrA2c8+Zw4IPHjoSG7e2YBAi36keq+S18fo3SZKBgBYsXJVQUFhQuJjM2amNWujqKqiqr0T EgAg3mJdmpiwKrFPrEGaFxv7et8+EeYAAMjNO6vnCn1URsYSW0RkZVXVhx9+JEoGQERB8k17XwE0 22HcuAmlZddTU8f3SUo6mpXVvK9f7P/SZPJbmpiQlZJ8ImXc4ZTxWSnJ34wcPiI0fMXKVc3dLl8p SE4e3bN7j5ra2uUrMkF/JWiF9q17YpJEQXa7eyckbNmyNTvb/t6G92zhYdOnz0hOTu7TJ7Gi4uax b76+mn+G1zV6VBUAICxszOTJw0cMLyoqsduzd+zYef7ChWdnz17yuyW/z8zc+J8bqWRkGmtdcbeV 1Wm9pE5Fw+qVK1PHjTuTl7t//4Gi4iKVsejIqPDIDiIRAMCpySIQAKhzOK6XlKicd4yJGT9hwoTx 40tLSzOWZFy6eJFKRsZaWRhtPQDQ71NcA6Zag0NmpqWlpqZG2Gwej7uqqrqystLZ5GSMIUFKBVEU wsPCY6KjrCEhTqczJydn08ebzufnA8BDav9QAHQQFIn+4A4AJpNf9x49Y2NjoyIjAwMD/f39KaVO p7PJ2VRdffvataslxUXV1VV3hgoS6KXeh5O2eOTTn8YAmCo/0I+JICKhmqY9vOp3Fm8DAPcmQwTU L5Y//YlzAGD3V8HaZs22BNAe0v7/L/SQ8ghAe8sjAO0tjwC0tzwC0N7yCEB7y/8AbBaT5ZPlRpoA AAAASUVORK5CYII='
        preserveAspectRatio='none' height='26.383' width='26.099' />
    </g>
</svg>