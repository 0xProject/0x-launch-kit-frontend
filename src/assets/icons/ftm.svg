<svg xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink"
width="26mm" height="26mm" viewBox="0 0 26 26" id="svg8">
    <g id="layer1" transform="translate(0 -271)">
        <image y="271.109" x="1.493" id="image4548" xlinkHref="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFMAAABuCAIAAADUNLgkAAAAA3NCSVQICAjb4U/gAAAGpElEQVR4 nO2dz0/iTBjH5Y162gQT8WoP/DhCssC1TQSOsom4R9mEcpVE2Csmlqu+ieUqB8oVE3GP1KR4XTYB ru2h/AMW/wL20PclRMrMdDozNYFvetI6fb5tP5155peB+Xy+s5H6x+8AfNPW+eZp63zztHXuhwaD QSQaFcvl6XTqw+Xnfsg0zUw2u7u3Zx+hoyNJkizLYhkDa+eWZVVrtYXn5SMcibQVhVkkTJ3Lshw6 OnK0vTgy2aymaQyCYeRc07RwJAL2vHyURNE0TaohUXf+AWn0gzb8FJ0DkF4cV9XqSSYDOIEe/LSc Q5EulUqL9/mp1wuHw4zhJ+8civRJJuNo4/7+/jAUYgY/Seej0QiMdDgcbrfbgBIsy7qqVtnAT8a5 ZVklUQREfBgK3SBHbJomGP7Q0ZF3+Ak4lyQJHWl0aZoGhj+ZSnmB35Pzp14PD2l0QeEvFAp48GM6 9440uqDw7+7tYcDv2jlZpNFFHH53zikhjS6C8LtwThtpdEHhv6pWoYW4cM4AaXSB4T/JZKAlkOiT CQR2AgEC5bi+rLeLot9m8NeVWV49R0gKUJ45Mef/feEo59WIeT6rt31JiqKk0ulGozGbzciWPJ1O s7lcNpcj1l2Jfr9X623AXSeYV6Pk+UyfeaVS6Xa73PGx42+n06koitlcbjAYYF9iZ2en2WxGYzFZ ltedkD89NXQdp2j0e//hvt5Ikv3zG0mikVdDkf6aTC6+qUy/cAvnc7tJWyoBonSVV0O77g5DoQ8t CN+c2xqNRh471aBIr0sKfHZuC7tTzVXX3Wd0bssV/Nhdd5/R+RwZfiJ5/udybgsKv1ukUWLzoQ23 qkQi8aKqgJrfUcWLiz/D4XW9fnBwQCkwRuPn3/J5wzDubm+DwSD4zHg8rvb7rVaL4ziqIbGbOTAe j59//Xp/fwefNplMXl9fiTf7V8XC+Ww2E8vlVDqN2JKVGo1oLKZ0OlSjou68YdtQFFd/NZvNRFFE v1kYoui89/wciUYlDxnreDzO5nLn5+c0JtJQcQ6NePU7D/jy23eQeM5P2DkU6WAwWK/Xh8Phh58X f/wwdJ3n+XUlE4efpHMo0uBamuO4F1VV+/11z58s/GScQ5HmeR6xlhYEAVzzk4Lfq3MUpFsPDy+q KggCerGVSsXQ9Xq9vu4EAvCjtIod28ZX1aqXATbELMA0zbOzM3DaI8sy04wFfEAH2FzlP5qmfU0m 0a/uT8aCjjS6BEH4Mxy2Hh6gzX50kXTOHR93u123SKOrWCyC4XclMs7tWtowjG/5PJEC1+ng4OC6 Xjd0PX966rEoAs7j8bih69eEHgWKOI57fHzsdrtesncCzieTyfn374znqCudTq1W89KeJfO223P0 az9/MsirB4NBNpcTRdHnlsyyZFmOxmLNZpNgmcuaTqdiuex9xMoW+YylWqtFolGyefVsNms0Gql0 2m2eDxC+c57n19WuZEd8lU4nlU6DkwKMYvGdC4IArl29ww9FetGCwCjc09uOUrviwQ9F2nsLggDn du2q9vvxeNzxBFfwoyBdvLjw3oIg9oWDNq1R4EdBevj7d6vVIjACgZ2rrcuuLMu6kSRwLrU6VFwo FKADbE+9HmJsfo6rQfNq9MPO813F5ue4GhR+RBFB2lF0Rxq85NUkkXYSi9Elt3k1d3ys9vsvqppI JOhFxWhEcVHzFy8uAKcFg8G721vDMCj1bSyL6SpsjuNarZba7zu2Ny8vLw1dr1QqbILxYf25IAgv qtp6eFiMKPA8b+j6v3d39OYJrGqX2ZU+qFgs5vN5udkUeJ7Bu70q35zv/A+/X1ff7jOxedo63zxt nW+ets43T/jOGQynIAovEnznzWaT5BoqXCmdTjQWw/lL7N6o5U41xls+2dI0LZlKYc+ZIDNbxJ6s 4sGFO5mmWSgUPM4WceEcOlklHInQXppqWZYE69i9qlZR3kHXew60223wApVMNktpaWpbUaALcdEv jbPPhN2jDvZPFn4A0oveeLevG/7eIqZpQhfoeIcfivRhKHR/f49Rstf9ZDRNg67OA4yNAEQQaUeR 2UOo3W5DV+eNRiMXBRJF2lHE9o1Cgb8kitBHRANpRxHeKwwFfmn9FFdKSDuKyv5wbuGnjbSjKO4J iAg/A6QdRXcfSBT4GSDtKBZ7f0Lhp420o9jt9wqFnx7SjmK9xy8YfkpIO8qHfZ0d4aeKtKP82dF6 vgQ/A6QdFZj7+r853t7evnz5sr+/z/7SPjv3Udte583T1vnmaet88/QXmguUkiRS4jcAAAAASUVO RK5CYII="
        preserveAspectRatio="none" height="25.891" width="21.866" />
    </g>
</svg>