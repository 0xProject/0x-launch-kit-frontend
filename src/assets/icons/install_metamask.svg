<svg width="61" height="56" viewBox="0 0 61 56" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path d="M57.1273 0L34.1201 17.4053L38.202 7.18916L57.1273 0Z" fill="#E2761B" />
    <path d="M3.69141 0L26.6985 17.7837L22.6166 7.18916L3.69141 0Z" fill="#E4761B" />
    <path d="M48.9634 40.4866L42.655 50.3244L56.014 54.1081L59.7248 40.865L48.9634 40.4866Z" fill="#E4761B" />
    <path d="M1.09399 40.865L4.80482 54.1081L18.1638 50.3244L11.8554 40.4866L1.09399 40.865Z" fill="#E4761B" />
    <path
        d="M17.4215 24.2155L13.7107 29.8911L26.6986 30.6479L26.3275 16.2695L17.4215 24.2155Z"
        fill="#E4761B"
    />
    <path d="M43.3974 24.2155L34.1204 16.2695V30.6479L47.1083 29.8911L43.3974 24.2155Z" fill="#E4761B" />
    <path d="M18.1636 50.3247L25.9563 46.5409L19.2768 40.8652L18.1636 50.3247Z" fill="#E4761B" />
    <path d="M34.8625 46.5409L42.6553 50.3247L41.542 40.8652L34.8625 46.5409Z" fill="#E4761B" />
    <path d="M42.655 50.3241L34.8623 46.5403L35.2334 51.4592V53.7294L42.655 50.3241Z" fill="#D7C1B3" />
    <path d="M18.1638 50.3236L25.5855 53.729V51.4587L25.9566 46.5398L18.1638 50.3236Z" fill="#D7C1B3" />
    <path d="M25.5853 37.46L18.9058 35.5681L23.7298 33.2979L25.5853 37.46Z" fill="#233447" />
    <path d="M35.2334 37.46L37.0888 33.2979L41.9129 35.5681L35.2334 37.46Z" fill="#233447" />
    <path d="M18.1636 50.3243L19.2769 40.4866L11.8552 40.8649L18.1636 50.3243Z" fill="#CD6116" />
    <path d="M41.542 40.4866L42.6552 50.3243L48.9636 40.8649L41.542 40.4866Z" fill="#CD6116" />
    <path
        d="M47.108 29.8921L34.1201 30.6488L35.2334 37.4596L37.0888 33.2975L41.9129 35.5677L47.108 29.8921Z"
        fill="#CD6116"
    />
    <path
        d="M18.9056 35.5677L23.7297 33.2975L25.5851 37.4596L26.6983 30.6488L13.7104 29.8921L18.9056 35.5677Z"
        fill="#CD6116"
    />
    <path d="M13.7109 29.8926L19.2772 40.8655L18.9061 35.5682L13.7109 29.8926Z" fill="#E4751F" />
    <path d="M41.9131 35.5677L41.542 40.865L47.1082 29.8921L41.9131 35.5677Z" fill="#E4751F" />
    <path
        d="M26.6987 30.6494L25.5854 37.4602L27.0698 45.7845L27.4409 34.8116L26.6987 30.6494Z"
        fill="#E4751F"
    />
    <path
        d="M34.1203 30.6494L33.3782 34.8116L33.7493 45.7845L35.2336 37.4602L34.1203 30.6494Z"
        fill="#E4751F"
    />
    <path
        d="M35.2336 37.4593L33.7493 45.7836L34.8625 46.5403L41.542 40.8647L41.9131 35.5674L35.2336 37.4593Z"
        fill="#F6851B"
    />
    <path
        d="M18.9058 35.5674L19.2768 40.8647L25.9563 46.5403L27.0696 45.7836L25.5852 37.4593L18.9058 35.5674Z"
        fill="#F6851B"
    />
    <path
        d="M35.2336 53.7303V51.4601L34.8625 51.0817H25.9566L25.5855 51.4601V53.7303L18.1638 50.325L20.7614 52.5952L25.9566 56.0006H34.8625L40.0577 52.5952L42.6553 50.325L35.2336 53.7303Z"
        fill="#C0AD9E"
    />
    <path
        d="M34.8625 46.5397L33.7493 45.783H27.0698L25.9565 46.5397L25.5854 51.4586L25.9565 51.0802H34.8625L35.2336 51.4586L34.8625 46.5397Z"
        fill="#161616"
    />
    <path
        d="M58.241 18.5405L60.0964 9.08105L57.1278 0L34.8628 17.027L43.3977 24.2161L55.6434 27.9999L58.241 24.9729L57.1278 23.8377L58.9832 22.3242L57.4989 21.1891L59.3543 19.6756L58.241 18.5405Z"
        fill="#763D16"
    />
    <path
        d="M0.7229 9.08105L2.57831 18.5405L1.46507 19.6756L3.32048 21.1891L1.83615 22.3242L3.69156 23.8377L2.57831 24.9729L5.17589 27.9999L17.4216 24.2161L25.9565 17.027L3.69156 0L0.7229 9.08105Z"
        fill="#763D16"
    />
    <path
        d="M55.6431 27.9993L43.3974 24.2156L47.1082 29.8912L41.542 40.8642H48.9636H59.7251L55.6431 27.9993Z"
        fill="#F6851B"
    />
    <path
        d="M17.4216 24.2156L5.17591 27.9993L1.09399 40.8642H11.8554H19.2771L13.7108 29.8912L17.4216 24.2156Z"
        fill="#F6851B"
    />
    <path
        d="M34.1203 30.6481L34.8624 17.0265L38.2022 7.18872H22.6167L25.9564 17.0265L26.6986 30.6481L27.0697 34.8102V45.7831H33.7492V34.8102L34.1203 30.6481Z"
        fill="#F6851B"
    />
</svg>
