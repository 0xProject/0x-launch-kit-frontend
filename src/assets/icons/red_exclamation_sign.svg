<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path
        d="M2.14713 14.0331L1.08647 12.9724L2.14713 14.0331L14.0331 2.14714C14.4474 1.73278 15.0094 1.5 15.5954 1.5L32.4046 1.5C32.9906 1.5 33.5526 1.73278 33.9669 2.14713L45.8529 14.0331C46.2672 14.4474 46.5 15.0094 46.5 15.5954L46.5 32.4046C46.5 32.9906 46.2672 33.5526 45.8529 33.9669L33.9669 45.8529C33.5526 46.2672 32.9906 46.5 32.4046 46.5H15.5954C15.0094 46.5 14.4474 46.2672 14.0331 45.8529L12.9724 46.9135L14.0331 45.8529L2.14714 33.9669C1.73278 33.5526 1.5 32.9906 1.5 32.4046L1.5 15.5954C1.5 15.0094 1.73278 14.4474 2.14713 14.0331Z"
        stroke="#FF6534"
        stroke-width="3"
    />
    <path d="M24.5 15L24.5 27.5" stroke="#FF6534" stroke-width="3" stroke-linecap="round" />
    <circle cx="24.5" cy="33.5" r="1.5" fill="#FF6534" />
</svg>
