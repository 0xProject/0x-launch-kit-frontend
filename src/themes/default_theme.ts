import { Theme, ThemeModalStyle, ThemeProperties } from './commons';

const modalThemeStyle: ThemeModalStyle = {
    content: {
        backgroundColor: '#fff',
        borderColor: '#dedede',
        bottom: 'auto',
        display: 'flex',
        flexDirection: 'column',
        flexGrow: '0',
        left: 'auto',
        maxHeight: '90%',
        minWidth: '350px',
        overflow: 'hidden',
        padding: '16px',
        position: 'relative',
        right: 'auto',
        top: 'auto',
    },
    overlay: {
        alignItems: 'center',
        backgroundColor: 'rgba(0, 0, 0, 0.4)',
        display: 'flex',
        justifyContent: 'center',
        zIndex: '12345',
    },
};

const lightThemeColors: ThemeProperties = {
    background: '#f4f3f4',
    backgroundERC721: '#F9FAFC',
    borderColor: '#dedede',
    boxShadow: '0 10px 10px rgba(0, 0, 0, 0.1)',
    buttonBuyBackgroundColor: '#3CB34F',
    buttonCollectibleSellBackgroundColor: '#00AE99',
    buttonConvertBackgroundColor: '#fff',
    buttonConvertBorderColor: '#dedede',
    buttonConvertTextColor: '#474747',
    buttonErrorBackgroundColor: '#FF6534',
    buttonPrimaryBackgroundColor: '#002979',
    buttonQuaternaryBackgroundColor: '#00AE99',
    buttonSecondaryBackgroundColor: '#474747',
    buttonSellBackgroundColor: '#FF6534',
    buttonTertiaryBackgroundColor: '#F6851B',
    buttonTextColor: '#fff',
    cardImageBackgroundColor: '#EBF0F5',
    cardBackgroundColor: '#fff',
    cardBorderColor: '#dedede',
    cardTitleColor: '#000',
    cardTitleOwnerColor: '#3CB34F',
    chartColor: '#00AE99',
    darkBlue: '#002979',
    darkGray: '#474747',
    darkerGray: '#666',
    dropdownBackgroundColor: '#fff',
    dropdownBorderColor: '#dedede',
    dropdownTextColor: '#000',
    errorButtonBackground: '#FF6534',
    errorCardBackground: '#FAF4EF',
    errorCardBorder: '#F39E4B',
    errorCardText: '#F68C24',
    ethBoxActiveColor: '#002979',
    ethBoxBorderColor: '#dedede',
    ethSetMinEthButtonBorderColor: '#000',
    ethSliderThumbBorderColor: 'rgba(0, 0, 0, 0.142)',
    ethSliderThumbColor: '#fff',
    gray: '#808080',
    green: '#3CB34F',
    iconLockedColor: '#000',
    iconUnlockedColor: '#C4C4C4',
    inactiveTabBackgroundColor: '#f9f9f9',
    lightGray: '#B9B9B9',
    logoERC20Color: '#0029FF',
    logoERC20TextColor: '#000',
    logoERC721Color: '#00AE99',
    logoERC721TextColor: '#000',
    marketsSearchFieldBackgroundColor: '#eaeaea',
    marketsSearchFieldBorderColor: '#dedede',
    marketsSearchFieldTextColor: '#333',
    modalSearchFieldBackgroundColor: '#fff',
    modalSearchFieldBorderColor: '#fff',
    modalSearchFieldPlaceholderColor: '#DEDEDE',
    modalSearchFieldTextColor: '#000',
    myWalletLinkColor: '#333333',
    notificationActive: '#F8F8F8',
    notificationIconColor: '#474747',
    notificationsBadgeColor: '#ff6534',
    numberDecimalsColor: '#dedede',
    red: '#FF6534',
    rowActive: '#FBFDFF',
    rowOrderActive: '#dedede',
    simplifiedTextBoxColor: '#F9F9F9',
    stepsProgressCheckMarkColor: '#fff',
    stepsProgressStartingDotColor: '#000',
    stepsProgressStepLineColor: 'rgba(0, 0, 0, 0.1)',
    stepsProgressStepLineProgressColor: '#000',
    stepsProgressStepTitleColor: '#e6e6e6',
    stepsProgressStepTitleColorActive: '#000',
    tableBorderColor: '#dedede',
    tdColor: '#000',
    textColorCommon: '#000',
    textDark: '#666',
    textInputBackgroundColor: '#F9F9F9',
    textInputBorderColor: '#dedede',
    textInputTextColor: '#000',
    textLight: '#999',
    textLighter: '#666',
    thColor: '#B9B9B9',
    tooltipBackgroundColor: '#222',
    tooltipTextColor: '#fff',
    topbarBackgroundColor: '#fff',
    topbarBorderColor: '#dedede',
    topbarSeparatorColor: '#dedede',
};

export class DefaultTheme implements Theme {
    public componentsTheme: ThemeProperties;
    public modalTheme: ThemeModalStyle;
    constructor() {
        this.componentsTheme = lightThemeColors;
        this.modalTheme = modalThemeStyle;
    }
}
