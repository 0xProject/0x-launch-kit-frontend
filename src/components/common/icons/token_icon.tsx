import React from 'react';
import ReactSVG from 'react-svg';
import styled, { withTheme } from 'styled-components';

import { Theme } from '../../../themes/commons';

interface Props {
    symbol: string;
    primaryColor?: string;
    isInline?: boolean;
    icon?: string;
    theme: Theme;
}

const IconContainer = styled.div<{ color: string; isInline?: boolean }>`
    align-items: center;
    background-color: ${props => (props.color ? props.color : 'transparent')};
    border-radius: 50%;
    display: ${props => (props.isInline ? 'inline-flex' : 'flex')};
    height: 26px;
    justify-content: center;
    width: 26px;
`;

const Label = styled.label`
    color: #fff;
    font-size: 0.7em;
    font-weight: 500;
    line-height: normal;
    margin: 0;
`;

const TokenIconContainer = (props: Props) => {
    const { symbol, primaryColor, theme, icon, ...restProps } = props;
    const fallBack = <Label>{symbol && symbol.toUpperCase()}</Label>;
    const Icon =
        // tslint:disable-next-line:jsx-no-lambda
        icon ? <ReactSVG src={icon as string}  fallback={() => fallBack} /> : fallBack;
    return (
        <IconContainer color={primaryColor || theme.componentsTheme.gray} {...restProps}>
            {Icon}
        </IconContainer>
    );
};

const TokenIcon = withTheme(TokenIconContainer);

export { TokenIcon };
