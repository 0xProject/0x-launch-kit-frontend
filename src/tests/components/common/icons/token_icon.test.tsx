import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';

import { TokenIcon } from '../../../../components/common/icons/token_icon';
import { DefaultTheme } from '../../../../themes/default_theme';

const theme = new DefaultTheme();

describe('TokenIcon', () => {
    it('TokenIcon ZRX to match snapshot', () => {
        // given
        const symbol = 'zrx';
        const primaryColor = '#232332';

        const tokenIcon = (
            <ThemeProvider theme={theme}>
                <TokenIcon symbol={symbol} primaryColor={primaryColor} />
            </ThemeProvider>
        );

        // when
        const tree = renderer.create(tokenIcon).toJSON();

        // then
        expect(tree).toMatchSnapshot();
    });

    it('TokenIcon MLN to match snapshot', () => {
        // given
        const symbol = 'mln';
        const primaryColor = '#232332';

        const tokenIcon = (
            <ThemeProvider theme={theme}>
                <TokenIcon symbol={symbol} primaryColor={primaryColor} />
            </ThemeProvider>
        );

        // when
        const tree = renderer.create(tokenIcon).toJSON();

        // then
        expect(tree).toMatchSnapshot();
    });

    it('Without token', () => {
        // given
        const symbol = 'test';
        const primaryColor = '#232332';

        const tokenIcon = (
            <ThemeProvider theme={theme}>
                <TokenIcon symbol={symbol} primaryColor={primaryColor} />
            </ThemeProvider>
        );

        // when
        const tree = renderer.create(tokenIcon).toJSON();

        // then
        expect(tree).toMatchSnapshot();
    });
});
