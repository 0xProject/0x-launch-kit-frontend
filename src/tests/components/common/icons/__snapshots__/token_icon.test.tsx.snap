// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TokenIcon TokenIcon MLN to match snapshot 1`] = `
<div
  className="sc-bdVaJa erQgzz"
  color="#232332"
>
  <label
    className="sc-bwzfXH jgqeEE"
  >
    MLN
  </label>
</div>
`;

exports[`TokenIcon TokenIcon ZRX to match snapshot 1`] = `
<div
  className="sc-bdVaJa erQgzz"
  color="#232332"
>
  <label
    className="sc-bwzfXH jgqeEE"
  >
    ZRX
  </label>
</div>
`;

exports[`TokenIcon Without token 1`] = `
<div
  className="sc-bdVaJa erQgzz"
  color="#232332"
>
  <label
    className="sc-bwzfXH jgqeEE"
  >
    TEST
  </label>
</div>
`;
